core/app/flags.go

```diff
@@ -24,6 +24,7 @@ type (
 		Analytics   string `help:"_If non-empty enable analytics using the specified user-id"`
 		CrashReport bool   `help:"_Automatically send crash reports to Google"`
 		DecodeStack string `help:"_Decode a stackdump generated by this executable"`
+		HelpFull    bool   `help:"_Display the full help"`
```

Comment:
Suggest `FullHelp` or `HelpVerbose` to avoid reading like helpful?
Also: Should this be hidden? Bit of a chicken-and-egg here.

Reply:
The `-fullhelp` flag, like the built-in `-help` flag is not shown in the help text. It is referred to, though, by the message if there are omitted flags.